src/components/ResultBar.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import OptionSwitcher from '@/components/OptionSwitcher.vue'
+import { ref } from 'vue'
+
+const sortByItems = ref([{ name: 'Release date' }, { name: 'Rating' }])
+const sortByDefaultValue = ref('Release date')
```

Comment:
if sortByItems property is using just to list the options and you don't need reactivity in this case then you can assign an object, without ref() or reactive() wrapper. Just optimisation stuff